modules/jwplayerBidAdapter.js

```diff
@@ -50,4 +85,144 @@ export const spec = {
   // onBidderError: function({ error, bidderRequest }) {}
 };
 
+function buildRequest(bidRequest, bidderRequest) {
+  // bidRequest.mediaTypes.video
+  // bidRequest.params (bid parameters)
+
+  // bidderRequest.gdprConsent
+  // bidderRequest.uspConsent
+  
+  // Open RTB Request Object
+  const openrtbRequest = {
+    id: params.bidId,
+    imp: buildRequestImpression(bidRequest, bidderRequest),
+    site: buildRequestSite(),
+    device: buildRequestDevice()
+  };
+
+  // Attaching GDPR Consent Params
+  if (bidderRequest.gdprConsent) {
+    deepSetValue(openrtbRequest, 'user.ext.consent', bidderRequest.gdprConsent.consentString);
+    deepSetValue(openrtbRequest, 'regs.ext.gdpr', (bidderRequest.gdprConsent.gdprApplies ? 1 : 0));
+  }
+
+  // CCPA
+  if (bidderRequest.uspConsent) {
+    deepSetValue(openrtbRequest, 'regs.ext.us_privacy', bidderRequest.uspConsent);
+  }
+
+  return JSON.stringify(openrtbRequest);;
+}
+
+function buildRequestImpression(bidRequest, bidderRequest) {
+  const impressions = [];
+
+  const impressionObject = {
+    id: bidRequest.adUnitCode,
+    secure: isSecure() ? 1 : 0
+  };
+
+  impressionObject.video = buildImpressionVideo(bidRequest);
+
+  const bidFloorData = buildBidFloorData(bidRequest);
+  impressionObject.bidfloor = bidFloorData.floor;
+  impressionObject.bidfloorcur = bidFloorData.currency;
+  
+  impressionObject.ext = buildImpressionExtension(); // TODO: Complete
+
+  impressions.push(impressionObject);
+
+  return impressions;
+}
+
+function buildImpressionVideo(bidRequest) {
+  const videoAdUnit = deepAccess(bidRequest, 'mediaTypes.video', {});
+
+  const playerSize = videoAdUnit.playerSize;
+
+  const contentWidth = playerSize[0][0];
+  const contentHeight = playerSize[0][1];
```

Comment:
since `playerSize` is not oRTB let's not support it. Instead we expect the dev to set values to `w` and `h`